src/main/java/uk/gov/di/ipv/cri/common/library/service/SessionService.java

```diff
@@ -181,4 +182,19 @@ public SessionItem getSessionByAuthorisationCode(String authCode)
 
         return sessionItem;
     }
+
+    // may be needed for tracking attempts
+    public void incrementAttemptCount(String sessionID) {
```

Comment:
if it is uncertain whether the `incrementAttemptCount` and `setLatestDcsResponseResourceId` then it makes sense to remove the methods and add them when it is understood that they are indeed required

Reply:
Good point, I'm going to leave in the incrementAttemptCount in and remove the setLatestDcsResponseResourceId(), after catching up with @smsgds he is going to implement the attemptCount method and branch off of this to do so

Reply:
makes sense, leaving in incrementAttemptCount so that @smsgds can use it in his work, for tracking attempts 😃